src/helpers/validationHelpers.js

```diff
@@ -52,11 +52,13 @@ export function getSelectionsFromChapterAndVerseCombo(bookId, chapter, verse, pr
     if (quote) {
       sorted = sorted.filter((filename) => {
         const currentSelectionsObject = fs.readJsonSync(path.join(selectionsPath, filename));
-        return currentSelectionsObject.contextId.quote === quote;
+        return isEqual(currentSelectionsObject.contextId.quote, quote); // add support for quote arrays
```

Comment:
Need to add support for string quotes as well

Reply:
What use case are you thinking that isEqual will not work with?  Since checks will be migrated it looks like any difference in type would fail to match as expected.

Reply:
Never mind didn't realize that deep-equal supported non-objects arguments